manifests/install.pp

```diff
@@ -5,8 +5,14 @@
 
   if $qpid::ensure == 'absent' {
     $_package_ensure = 'purged'
+    $_tools_ensure   = $qpid::ensure
```

Comment:
Would we not want to use purge here too?

Reply:
I felt it was more consistent to have `class qpid::tools( Enum['present', 'absent'] $ensure = 'present', )` similar to the other clases